src/views/virtualmachinesinstance/details/tabs/details/components/Details/CreateAt/CreateAt.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Trans } from 'react-i18next';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  DescriptionListDescription,
+  DescriptionListTermHelpText,
+  DescriptionListTermHelpTextButton,
+  Popover,
+} from '@patternfly/react-core';
+
+type CreateAtProps = {
+  createat?: string;
+};
+
+const CreateAt: React.FC<CreateAtProps> = () => {
+  const { t } = useKubevirtTranslation();
+  return (
+    <>
+      <DescriptionListTermHelpText>
+        <Popover
+          hasAutoWidth
+          maxWidth="30rem"
+          headerContent={t('Created At')}
+          bodyContent={
+            <Trans ns="'plugin__kubevirt-plugin'">
```

Comment:
is it on purpose with double and single quotes? maybe saving it as a const?

Reply:
fixed